### DISCUSSION TITLE
OS X El Capitan Ver 10.11.6

### DISCUSSION THREAD

**Discussion by storeilly** [jamulussoftware/jamulus#2246] [Bug Investigation]
Is there any known issues with installing **jamulus_3.8.1_mac_legacy** on OS X 10.11.6?

I just tried to install a fresh download of **jamulus_3.8.1_mac_legacy.dmg** it appeared to install ok but it would not run, giving an error message that a newer version of OS (10.13 I think) was required.

The iMac is 3.06 GHz Intel Core i3 with 4GB RAM

I have the same error from **jamulus_3.8.1.dmg** and **jamulus_3.8.0.dmg** which I would expect but **jamulus_3.8.0_mac_legacy.dmg** works fine. Any ideas?

**Comment by ann0see** [comment]
Yes. That’s a known (fixed) bug. You should be able to use the artifact from the last build (check the Actions tab on GH).
@jamulussoftware/maindevelopers we might want to release a nightly or even 3.8.2 with only this bug fix

**Reply by pljones** [reply]
So that was #2223, which went in yesterday.

We're looking at this lot since 3.8.1:

ccecafe8 2022-01-14 Merge pull request #2223 from softins/macos-legacy [Tony Mountifield]
5eec44f6 2022-01-13 Merge pull request #2237 from softins/android-update [Tony Mountifield]
bc258cee 2022-01-10 Merge pull request #2231 from henkdegroot/server-tray-icon-update [ann0see]
ade9f6e6 2022-01-10 Merge pull request #2201 from henkdegroot/windows-uninstall-details [ann0see]
bf7dd989 2022-01-10 Merge pull request #2215 from henkdegroot/hide-asio-button-withjack [Tony Mountifield]
f053f366 2022-01-10 Merge pull request #2187 from henkdegroot/version-details [Tony Mountifield]
ff281ca1 2022-01-08 Merge pull request #2225 from henkdegroot/Improve-ps-url-redirect [Peter L Jones]
f83c83c4 2022-01-08 Merge pull request #2208 from ann0see/upgrade/updateNsis [Peter L Jones]
29ede6b8 2022-01-07 Merge pull request #2221 from henkdegroot/pr-template-update [Peter L Jones]
26a896d2 2021-12-29 Merge pull request #2196 from softins/remove-submodules [Tony Mountifield]
e1e6d8e1 2021-12-28 Merge pull request #2177 from pljones/patch/ocd-reorder-of-serverdlg-bits [ann0see]
f1ff6833 2021-12-20 Merge pull request #2149 from henkdegroot/detect-asio-install [Peter L Jones]
0d61e69c 2021-12-18 Merge pull request #2085 from BLumia/i18n-fixes-382 [Peter L Jones]
78a5b196 2021-12-17 Merge pull request #2173 from pljones/bugfix/use-default-server-name-in-mixer [Peter L Jones]
48e321d6 2021-12-17 Merge pull request #2158 from softins/vibraphone-icon [Tony Mountifield]
081cce4e 2021-12-11 Merge pull request #2151 from softins/channel-allocation [Tony Mountifield]
f9951592 2021-12-07 Merge pull request #2124 from softins/qt-regex-update [Tony Mountifield]
d0a16b8f 2021-12-06 Merge pull request #2146 from atsampson/protcol [Tony Mountifield]
eb9d161a 2021-11-26 Merge pull request #2137 from rolamos/fixing_issue2123 [Tony Mountifield]
c633a543 2021-11-26 Merge pull request #1809 from ngocdh/ownfader1st [Tony Mountifield]
51589960 2021-11-13 Merge pull request #1994 from jamulussoftware/djfun-devel/unicode-name [Peter L Jones]
9aac3639 2021-11-13 Merge pull request #2120 from softins/fix-feedback-detection-state [Tony Mountifield]
5c21445a 2021-11-08 Merge pull request #1688 from henkdegroot/MeterStyleSelection [Peter L Jones]
031fdd85 2021-10-25 Merge pull request #2080 from DavidSavinkoff/master [Tony Mountifield]

**Reply by pljones** [reply]
There's a few UI changes in there.

@gilgongo given what's there, would there need to be a full release, including updating the website docs?

And we'd have to update translations in places, too - I know those will have changed for several of the new bits above.

I'm not keen on patching 3.8.1 with just the one change - branching for 3.8.2, that is... partly because we've not done it on Jamulus and partly because I'm not a fan of backporting.

**Reply by gilgongo** [reply]
The meter style selection and server dialogue screens would be nice to have in the docs. But if the value of releasing the macOS fix is big enough, we can have the docs (and translations) lag for a while. It might take a few weeks for all translations to come in perhaps before we can do a site release to get it in line.

**Reply by softins** [reply]
Just doing the one change for MacOS Legacy would mean 3.8.2 is the same as 3.8.1 for all other platforms, which doesn't sound like it's worth doing. Just looking at the changes since 3.8.1 (`git log --graph master...r3_8_1`) a lot of them appear to be fixes for either bugs or sub-optimal things, and nothing really new or groundbreaking (such as the JSON-RPC stuff would be).

So we could do a freeze and make a 3.8.2, with some benefit. Are there any more under-the-hood things that should be included in that? What about @pljones NBT changes? I think maybe do a 3.8.2 before including any of the open PRs, such as #2199

**Reply by pljones** [reply]
> What about @pljones NBT changes? 

Might be 4.x.x ...  Progress is being made but it makes me want to ditch some backwards compatibility...

Anyway, yes, I think 3.8.2 before #2199 might be good, too.  I definitely agree there's a lot in that list that would be good to release pretty soon, as there are fixes.

**Comment by ann0see** [comment]
@storeilly just tagged a nightly: https://github.com/jamulussoftware/jamulus/releases/tag/r3.8.1devNightly1

It's also interesting to see what happens if the changelog is empty (i.e. what npostavs changes do in a release)
